GOVERNANCE.md

```diff
@@ -0,0 +1,100 @@
+
+Some contributors will already be engaging with the project as users, but will also find themselves doing one or more of the following:
+
+- supporting new users (current users often provide the most effective new user support),
+- reporting bugs,
+- identifying requirements,
+- supplying graphics and web design,
+- programming,
+- assisting with project infrastructure,
+- writing documentation,
+- fixing bugs,
+- adding features.
+  
+As contributors gain experience and familiarity with the project, they may find that the project lead starts relying on them more and more. When this begins to happen, they gradually adopt the role of maintainer, as described above.
+
+### Users
+
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements.
+
+Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+- evangelizing about the project,
+- informing developers of project strengths and weaknesses from a new user’s perspective,
+- providing moral support (a ‘thank you’ goes a long way),
+- providing financial support.
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described above.
+
+### Support
+
+All participants in the community are encouraged to provide support for new users within the project management infrastructure. This support is provided as a way of growing the community. Those seeking support should recognise that all support activity within the project is voluntary and is therefore provided as and when time allows. A user requiring guaranteed response times or results should therefore seek to purchase a support contract from a vendor. (Of course, that vendor should be an active member of the community.) However, for those willing to engage with the project on its own terms, and willing to help support other users, the community support channels are ideal.
+
+### Contribution Process
+
+Anyone can contribute to the project, regardless of their skills, as there are many ways to contribute. For instance, a contributor might be active on the project mailing list and issue tracker, or might supply patches. The various ways of contributing are described in more detail in our roles in open source document.
+
+The developer mailing list is the most appropriate place for a contributor to ask for help when making their first contribution.
```

Comment:
```suggestion
The developer mailing list is the most appropriate place for a contributor to ask for help when making their first contribution.
```
Maybe in our case we can "redirect" them to Slack channel?
Probably we will set up a mailing list too, but Slack is more convenient, at least in my opinion.

Reply:
Good idea - I updated the document.